### ISSUE TITLE
File logger not working for plugins

### ISSUE THREAD

**Issue by aacuevas** [open-ephys/plugin-GUI#549]
While all LOGx commands in the source log into the console, only those called by the main GUI write into activity.log, while the ones called by plugins fails to do so.

My guess is that since the Logger class is in a header file (Utils.h) and referenced through a static method, this class is not crossing binary boundaries, but it's being created once per each plugin. But since only the one in the main GUI is being initialized with the filename, only main GUI events are being logged.

**Comment by medengineer** [comment]
You are correct, each binary is generating its own instance of the OELogger and the plugins can't see the log path assigned to the main GUI. 

I'm not seeing a quick fix here without breaking the PLUGIN_API so we plan to refactor this with a more robust solution before 1.0 release.

Workaround for now is to just pipe the console output to a file manually via terminal as needed.
